sklearn/feature_extraction/tests/test_dict_vectorizer.py

```diff
@@ -26,10 +25,10 @@ def test_dictvectorizer():
                     v = DictVectorizer(sparse=sparse, dtype=dtype, sort=sort)
                     X = v.fit_transform(iter(D) if iterable else D)
 
-                    assert_equal(sp.issparse(X), sparse)
-                    assert_equal(X.shape, (3, 5))
-                    assert_equal(X.sum(), 14)
-                    assert_equal(v.inverse_transform(X), D)
+                    assert sp.issparse(X) == sparse
```

Comment:
Although we have a different convention for new code, updating existing code makes this PR hard to review

Reply:
then don't..

Reply:
In this case it's not terribly bad, cv which is why I didn't suggest reverting, but for future reference it's better to keep a PR focused on one issue